lxd/container.go

```diff
@@ -738,6 +720,7 @@ func containerCreateInternal(d *Daemon, args containerArgs) (container, error) {
 	args.CreationDate = dbArgs.CreationDate
 	args.LastUsedDate = dbArgs.LastUsedDate
 
+	// Setup the container struct and finish creation (storage and idmap)
```

Comment:
I meant that comment to be a pointer for anyone wondering why we're not doing storage stuff there to point that the function call after that comment does that.